shiny/_docstring.py

```diff
@@ -29,6 +30,25 @@ def no_example(func: F) -> F:
     return func
 
 
+def no_example_express(decorator: Callable[..., F]) -> F | Callable[..., F]:
+    """
+    Prevent ``@add_example()`` from throwing an error about missing Express examples.
+    """
+
+    @wraps(decorator)
+    def wrapper_decorator(*args: Any, **kwargs: Any) -> F:
+        try:
+            # Apply the potentially problematic decorator
+            return decorator(*args, **kwargs)
+        except ExpressExampleNotFoundException:
+            # If an error occurs, return the original function
+            if args and callable(args[0]):
+                return args[0]
+            raise
+
+    return wrapper_decorator
```

Comment:
Ideally, instead of adding another decorator, it seems better if `@no_example` took a `mode`? Like `@no_example(mode = "express")`

That said, given the timeline on this, I'd be fine with how it is now, especially if not gonna have a need for it relatively soon. One thing I ask is that, if we plan on doing something like what I propose above, let's add a TODO/NOTE about it in `check_if_missing_expected_example`?

Reply:
Great suggestion! I updated `@no_example()` to be used like this:

```python
@add_example()
@no_example("express")
def a_core_thing_only():
    pass
```

A couple caveats/notes:

* You now need to call `@no_example()` where before we used `@no_example`
* `@no_example()` needs to come _before_ any `@add_example()` decorators
* `@no_example()` alone ignores both, or `"express"` or `"core"` to ignore individually. (You can stack these.)
* `@no_example()` needs to be called without the named mode arg. (We can address this later if its an issue; in short its because we're looking at literal strings in quartodoc and they need to be `no_example()`, `no_example("express")` or `no_example("core")`.)